Cargo.toml

```diff
@@ -35,3 +35,8 @@ proc-macro2 = "1.0"
 quote = "1.0.9"
 syn = { version = "1.0.69", features = ["full"] }
 proc-macro-crate = "0.1.5"
+
+# Required to build tests with near-sdk v4.1.1, see
+# TODO(): Remove after upgrading to near-sdk v5.
+[patch.crates-io]
+parity-secp256k1 = {git = "https://github.com/paritytech/rust-secp256k1"}
```

Comment:
I would recommend adding a hash commit because, in such a way, you download the latest code from the master branch which potentially could be broken.

Reply:
Good point! I’ve added a `tag` corresponding to the last release of the `parity-secp256k1` crate.

Would you mind pointing that out [here](https://github.com/near/near-sdk-rs/issues/1126) as well? The workaround recommended to the community is currently missing a `tag` (or anything else that pins the downloaded code).